.cirrus.yml

```diff
@@ -1,41 +0,0 @@
-build_linux_x86_64_task:
-    container:
-        dockerfile: ci/Dockerfile
-    autotools_script:
-        - aclocal
-        - autoconf
-        - autoheader
-        - automake --add-missing
-    configure_script:
-        - ./configure
-    make_script:
-        - make
-
-build_linux_i386_task:
-    container:
-        dockerfile: ci/Dockerfile
-    autotools_script:
-        - aclocal
-        - autoconf
-        - autoheader
-        - automake --add-missing
-    configure_script:
-        - CFLAGS="-m32" ./configure
-    make_script:
-        - make
-
-build_freebsd_x86_64_task:
```

Comment:
FreeBSD (as a proxy for other *BSDs) was the main reason I was using cirrus. Is it possible to cover this 'testing' with GitHub Actions?

Reply:
LGTM, thanks!